lane_change:
    weight: 10.0 # Bezier Weight 1.3
    length_lane_changed: 30

bezier:
  size: 60

push:
    length: 13.0
    bezier_length: 14.0

weights:
    weight_lane: 1.7 # 2,3차선 Weight 1.7
    atypical_weight: 2.3 # 비정형 Weight 1.7  2.3

debug:
  num: 3
